src/category_theory/sites/sheaf_of_types.lean

```diff
@@ -268,6 +268,64 @@ begin
   exact t.restrict (le_generate R),
 end
 
+namespace family_of_elements
```

Comment:
The rest of this file never works within the namespace `family_of_elements` but instead prefixes the relevant lemmas with `family_of_elements.<name>`. For consistency, maybe do this here as well?